botCommands/debug.test.js

```diff
@@ -63,18 +63,18 @@ describe('/debug', () => {
 
 describe('/debug', () => {
   it('should return the correct output', () => {
-    expect(command.cb({ content: '/debug' })).toMatchSnapshot();
+    expect(command.cb()).toMatchSnapshot();
   });
 });
 
 describe('/debug js', () => {
-  it('should return the correct output', () => {
+  xit('should return the correct output', () => {
     expect(command.cb({ content: '/debug js' })).toMatchSnapshot();
   });
 });
 
 describe('/debug rb', () => {
-  it('should return the correct output', () => {
+  xit('should return the correct output', () => {
```

Comment:
Let's just remove these since they are no longer needed. We also should remove similar test cases in the `regex` describe block, where we check for `debug js` and `debug rb`.